pkg/erofs/verity.go

```diff
@@ -0,0 +1,5 @@
+package erofs
+
+func verityDataLocation(sblock *superblock) (uint64, error) {
+	return uint64((1 << sblock.BlockSizeBits) * sblock.Blocks), nil
```

Comment:
do we know for sure that erofs does not append any data past the superblock?

Reply:
https://docs.kernel.org/filesystems/erofs.html

Reply:
The superblock is fixed size; presumably the blocksize * blocks will be total image size;   but it's not entirely clear from the link, nor the more detailed link here:

https://erofs.docs.kernel.org/en/latest/core_ondisk.html

I guess we'll find out =)